lib/types.ts

```diff
@@ -316,6 +316,23 @@ export interface SpaceAPI {
   ) => Promise<ScheduledAction[]>
   /* Returns a list of scheduled actions for the currenst space & environment */
   getAllScheduledActions: () => Promise<ScheduledAction[]>
+
+  signRequest?: (request: CanonicalRequest) => Promise<AppSignedRequest>
```

Comment:
Alternatively I could make it return just the `additionalHeaders` instead of wrapping it together with the unneeded `sys`. This would also save us from redefining the type for the `AppSignedRequest` entity here.

Reply:
I would do so, especially given the fact that there's more in `sys` compared to what you've defined below.